src/app/home-page/widget/news-box/news-box.component.html

```diff
@@ -2,6 +2,7 @@
   <div fxLayout="column" fxLayoutAlign="start stretch" class="h-100">
     <mat-card-header>
       <mat-card-title>News & Updates</mat-card-title>
+      <span class="bubble">4</span>
```

Comment:
Is it ever possible to have a number of items which is not 4?

Reply:
I think this is matching https://github.com/dockstore/extra-content/blob/develop/news.html 
And I do think this may be a bit unwieldy if we need to modify both places when adding/removing news. 

If there's not a simple way to count the number of news items, we may need to skip this (even if it does look better) for usability.

Reply:
> Is it ever possible to have a number of items which is not 4?

Mm yes, I was thinking about that and it's a good point, I assumed we always maintained four at a time...but that may not always be valid.

I'm adding in a check to count the number of `news-entry` classes in the HTML string retrieved, which will indicate the number of news articles.

Ex. with two articles returned:
![image](https://user-images.githubusercontent.com/97123241/229565581-01279bbf-8b4c-4fbe-a330-d04f802617d8.png)